lxd/devlxd_storage.go

```diff
@@ -349,7 +349,7 @@ func devLXDStoragePoolVolumePutHandler(d *Daemon, r *http.Request) response.Resp
 
 	err = addStoragePoolVolumeDetailsToRequestContext(d.State(), req)
 	if err != nil {
-		response.DevLXDErrorResponse(err)
+		return response.DevLXDErrorResponse(err)
```

Comment:
this should be added to the original commit that added `devLXDStoragePoolVolumePutHandler`